### ISSUE TITLE
[Feature Request] M3U support

### ISSUE THREAD

**Issue by TiBeN** [libretro/fuse-libretro#123]
Like in others computer emulation cores, M3U would be a great improvement. I have a bunch of multi sided tzx games i can't use because i can't switch media at runtime.

This has been already reported at #72, i don't know why this issue has been closed seens this FR has not been adressed.

**Comment by MartinMSPedersen** [comment]
Yes. I would be nice with multitape support - also for the ZX81 core

**Comment by yonailo** [comment]
+1

**Comment by walterg74** [comment]
+2

**Comment by yonailo** [comment]
M3u is not needed because you can concatenate all files (tzx or tap) into a single file.
